### PR TITLE
OCM-7288 | Add management parameters to NodePool model

### PR DESCRIPTION
**Pull request by philipwu08**

### CODE REVIEW

**Comment by lucasponce** [comment]
/hold

Please, let's put this code on-hold, there are open discussions about the feature.

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.375 May 26 2024
+- Add `ManagementUpgrade` parameters to the `NodePool` model to support additional upgrade configuration.
```
Comment (gdbranco) [inline CHANGES.md:6]: Please include as well
- Rename file for `access_request_status_type` to .model

File: CHANGES.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.375 May 26 2024
```
Comment (lucasponce) [inline CHANGES.md:5]: Code looks ok, but the CHANGES.md needs to capture other MRs merged

Comment (mikeshng) [inline CHANGES.md:5]:
```suggestion
## 0.0.375 May 30 2024
- Add `AdditionalAllowedPrincipals` to `AWS` type to support additional allowed principal ARNs to be added to the hosted control plane's VPC Endpoint Service.
```

**Review by lucasponce — APPROVED** [review]

**Pull request merged**